src/server/JasmineGraphInstanceService.cpp

```diff
@@ -80,4301 +144,5243 @@ void *instanceservicesession(void *dummyPt) {
-
-            char dataBuffer[INSTANCE_LONG_DATA_LENGTH + 1];
-            bzero(dataBuffer, INSTANCE_LONG_DATA_LENGTH + 1);
-            read(connFd, dataBuffer, INSTANCE_LONG_DATA_LENGTH);
-            string hostList = (dataBuffer);
-            instance_logger.log("Received Hosts List: " + hostList, "info");
-
-            // Put all hosts to a map
-            std::map<std::string, JasmineGraphInstanceService::workerPartitions> graphPartitionedHosts;
-            std::vector<std::string> hosts = Utils::split(hostList, '|');
-            int count = 0;
-            int totalPartitions = 0;
-            for (std::vector<std::string>::iterator it = hosts.begin(); it != hosts.end(); ++it) {
-                if (count != 0) {
-                    std::vector<std::string> hostDetail = Utils::split(*it, ',');
-                    std::string hostName;
-                    int port;
-                    int dataport;
-                    std::vector<string> partitionIDs;
-                    for (std::vector<std::string>::iterator j = hostDetail.begin(); j != hostDetail.end(); ++j) {
-                        int index = std::distance(hostDetail.begin(), j);
-                        if (index == 0) {
-                            hostName = *j;
-                        } else if (index == 1) {
-                            port = stoi(*j);
-                        } else if (index == 2) {
-                            dataport = stoi(*j);
-                        } else {
-                            partitionIDs.push_back(*j);
-                            totalPartitions += 1;
-                        }
-                    }
-                    graphPartitionedHosts.insert(pair<string, JasmineGraphInstanceService::workerPartitions>(
-                        hostName, {port, dataport, partitionIDs}));
-                }
-                count++;
-            }
-            /*Receive file*/
-            write(connFd, JasmineGraphInstanceProtocol::SEND_FILE_NAME.c_str(),
-                  JasmineGraphInstanceProtocol::SEND_FILE_NAME.size());
-            instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::SEND_FILE_NAME, "info");
+        if (sockfd < 0) {
+            std::cout << "Cannot create socket" << std::endl;
+            return;
+        }
 
-            bzero(data, INSTANCE_DATA_LENGTH + 1);
-            read(connFd, data, INSTANCE_DATA_LENGTH);
-            string fileName = (data);
-            instance_logger.log("Received File name: " + fileName, "info");
-            write(connFd, JasmineGraphInstanceProtocol::SEND_FILE_LEN.c_str(),
-                  JasmineGraphInstanceProtocol::SEND_FILE_LEN.size());
-            instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::SEND_FILE_LEN, "info");
+        server = gethostbyname(host.c_str());
+        if (server == NULL) {
+            std::cout << "ERROR, no host named " << server << std::endl;
```

Comment:
Lets replace `cout` with a logger

Reply:
Removed all occurrences of `std::cout` in next PR

Reply:
Ok